dcaf_case__management/Gemfile

```diff
@@ -0,0 +1,21 @@
+source 'https://rubygems.org'
+
+gem 'rails', '4.2.1'
```

Comment:
We'll want to bump rails version, though it's not esp urgent. Will make an issue for this.